src/components/ResourceListItemButton.tsx

```diff
@@ -15,49 +18,96 @@ const StyledTypography: OverridableComponent<TypographyTypeMap<unknown, 'span'>>
-      <Thumbnail
-        resourceIdentifier={resource.identifier}
-        alt={resource.features.dlr_title ?? t('resource.metadata.resource')}
-      />
-      <StyledListItemText
-        primary={`${resource.features.dlr_title} (${resource.features.dlr_content_type})`}
-        secondary={
-          <>
-            {showSubmitter && resource.features.dlr_submitter_email && (
-              <StyledTypography style={{ display: 'block' }} component="span" variant="body2" color="textPrimary">
-                {resource.features.dlr_submitter_email}
-              </StyledTypography>
-            )}
-            {showTimeCreated && resource.features.dlr_time_created && (
-              <StyledTypography style={{ display: 'block' }} component="span" variant="body2" color="textPrimary">
-                {resource.features.dlr_time_created}
-              </StyledTypography>
-            )}
-            {resource.features.dlr_identifier_handle && showHandle && (
-              <span>
-                {t('handle')}: {resource.features.dlr_identifier_handle}
-              </span>
-            )}
-          </>
-        }
-      />
-    </ListItem>
+    <StyledListItem data-testid={`list-item-resources-${resource.identifier}`}>
+      <Button component="a" href={`/resource/${resource.identifier}`}>
+        <Thumbnail
+          resourceIdentifier={resource.identifier}
+          alt={resource.features.dlr_title ?? t('resource.metadata.resource')}
+        />
+        <StyledListItemText
+          primary={`${resource.features.dlr_title} (${resource.features.dlr_content_type})`}
+          secondary={
+            <>
+              {showSubmitter && resource.features.dlr_submitter_email && (
+                <StyledTypography style={{ display: 'block' }} component="span" variant="body2" color="textPrimary">
+                  {resource.features.dlr_submitter_email}
+                </StyledTypography>
+              )}
+              {showTimeCreated && resource.features.dlr_time_created && (
+                <StyledTypography style={{ display: 'block' }} component="span" variant="body2" color="textPrimary">
+                  {resource.features.dlr_time_created}
+                </StyledTypography>
+              )}
+              {resource.features.dlr_identifier_handle && showHandle && (
+                <span>
+                  {t('handle')}: {resource.features.dlr_identifier_handle}
+                </span>
+              )}
+            </>
+          }
+        />
+      </Button>
+      {handleDelete && (
+        <>
+          <StyledDeleteButton
+            data-testid={`delete-my-resources-${resource.identifier}`}
+            color="secondary"
+            startIcon={<DeleteIcon fontSize="large" />}
+            size="large"
+            onClick={() => setShowConfirmDialog(true)}>
+            {t('common.delete')} {window.innerWidth < 600 && resource.features.dlr_title}
+          </StyledDeleteButton>
+          <ConfirmDeleteDialog
```

Comment:
vet ikke om det har noe å si for ytelsen - men burde kanskje hele elementet kun vært vist om showConfirmDialog=true ? Nå blir det liggende i dom'en for hver ressurs

Reply:
https://github.com/mui-org/material-ui/issues/4818

Virker som vi går glipp av lukke og åpne animasjonen hvis vi gjør som du sier. (Klarer riktignok ikke selv å se store forskjellen)